tests/scripts/DataLoaderTests.js

```diff
@@ -0,0 +1,191 @@
+/*!
+ * GPII Preferences Data Loader Tests
+ *
+ * Copyright 2016-2017 OCAD University
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = fluid || require("infusion"),
+    jqUnit = fluid.require("node-jqunit", require, "jqUnit"),
+    gpii = fluid.registerNamespace("gpii");
+
+require("../../scripts/shared/dataLoader.js");
+require("./DataLoaderTestsUtils.js");
+
+fluid.logObjectRenderChars = 10240;
+
+(function () {
```

Comment:
These tests don't run from the browser and so the file-scope closure is unnecessary